tests/instrumentation-framework/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>hawkular-apm</artifactId>
+    <groupId>org.hawkular.apm</groupId>
+    <version>0.9.3.Final-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>instrumentation-test-framework</artifactId>
+
+  <properties>
+    <!-- override some version for spotify docker client -->
+    <version.com.fasterxml.jackson.core>2.6.0</version.com.fasterxml.jackson.core>
+    <version.org.apache.httpcomponents>4.5</version.org.apache.httpcomponents>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>docker-client</artifactId>
+      <version>5.0.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.21</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.5</version>
+    </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-apm-agent-to-target-dir</id>
+            <phase>test-compile</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.hawkular.apm</groupId>
+                  <artifactId>hawkular-apm-agent</artifactId>
+                  <version>${project.version}</version>
+                  <type>jar</type>
+                </artifactItem>
```

Comment:
You will need to add <destFileName>hawkular-apm-agent.jar</destFileName> to make sure the jar name does not contain the version - this is necessary as the jar contains a boot path (in the manifest) to its own jar, used to configure the javaagent, and if the name is not the same it causes problems when executing some of the byteman rules (i.e. not finding some byteman exception classes)